contrib/checkbox-ce-oem/checkbox-provider-ce-oem/bin/serial_config_parser.py

```diff
@@ -19,8 +54,21 @@ def print_ports_config(string: str):
         serial = {}
         port_type, port_node, baud_rate = config_parts
         serial["type"] = port_type
+
+        # Init a config dict if type is RS485
+        if port_type == "RS485":
+            serial["rs485_conf"] = {
+                "rts_level_for_tx": True,
+                "rts_level_for_rx": False,
+                "delay_before_tx": 0.0,
+                "delay_before_rx": 0.0,
```

Comment:
May I know how do you decide these initial values?

Reply:
It's refer to serial.rs485 module. https://pyserial.readthedocs.io/en/latest/pyserial_api.html#serial.rs485.RS485Settings
I've put those information into the script as a comment for reference.